packages/workspace/src/generators/workspace/files/package.json__tmpl__

```diff
@@ -3,31 +3,7 @@
   "version": "0.0.0",
   "license": "MIT",
   "scripts": {
-    <% if(cli === 'angular') { %>
-    "ng": "nx",
-    "postinstall": "node ./decorate-angular-cli.js",
-    <% } %>
-    "nx": "nx",
-    "start": "<%= cliCommand %> serve",
-    "build": "<%= cliCommand %> build",
-    "test": "<%= cliCommand %> test",
-    "lint": "nx workspace-lint && <%= cliCommand %> lint",
-    "e2e": "<%= cliCommand %> e2e",
-    "affected:apps": "nx affected:apps",
-    "affected:libs": "nx affected:libs",
-    "affected:build": "nx affected:build",
-    "affected:e2e": "nx affected:e2e",
-    "affected:test": "nx affected:test",
-    "affected:lint": "nx affected:lint",
-    "affected:dep-graph": "nx affected:dep-graph",
-    "affected": "nx affected",
-    "format": "nx format:write",
-    "format:write": "nx format:write",
-    "format:check": "nx format:check",
-    "update": "nx migrate latest",
-    "workspace-generator": "nx workspace-generator",
-    "dep-graph": "nx dep-graph",
-    "help": "nx help"
+    "nx": "nx"
```

Comment:
Probably don't even need the `nx` script since `yarn nx` will run the `nx` binary in `node_modules`.  For npm users should  just use `npx nx`?

Reply:
Agree, the only reason maybe to leave it in is to have shorter npm scripts, without having to prefix with npx etc